### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Ben00793** [BJReplay/ha-solcast-solar#18] [General]
A massive thanks from me, for keeping this integration alive!

**Comment by lamixer** [comment]
Yes, Thank you!

**Comment by autoSteve** [comment]
This code will live on. @BJReplay FTW! Well done, bloke.

**Comment by wimdebruyn** [comment]
The increased level of activity for this integration is great to see.  Thank you. Upwards and onwards!
